### PR TITLE
Generate a Z_INDEX.md file with z indexes used in the project

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

**Comment by erwinmombay** [comment]
@sriramkrish85 PTAL

File: package.json [hunk 1]
```diff
@@ -1,99 +0,0 @@
-{
```
Comment (cramforce) [inline package.json]: ?

Comment (erwinmombay) [inline package.json]: woot, thats embarrassing. meant to reset to master, not revert/delete the file! undone.

File: build-system/tasks/get-zindex.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  rows.unshift.apply(rows, tableHeaders);
+  Object.keys(filesData).sort().forEach((fileName, fileIdx) => {
+    var selectors = filesData[fileName];
+    Object.keys(selectors).sort().forEach((selectorName, selectorIdx) => {
+      var zIndexes = selectors[selectorName];
+      var row = [selectorName, zIndexes.join(', '), fileName];
+      rows.push(row);
+    });
+  });
+  var tbl = table(rows, tableOptions);
```
Comment (sriramkrish85) [inline build-system/tasks/get-zindex.js]: could we sort by z-index before pushing?

Comment (erwinmombay) [inline build-system/tasks/get-zindex.js]: yeah, the reason why i didn't do that is its possible to have multiple z-indexes on a single row if for example one declared a z-index in one rule on the same file and then reassigned a new z-index somewhere down the file. (i know its stupid and we probably would catch it in review, but its just possible in css thats all). will sort

Comment (erwinmombay) [inline build-system/tasks/get-zindex.js]: done.

**Comment by erwinmombay** [comment]
@sriramkrish85 PTAL

**Comment by sriramkrish85** [comment]
@erwinmombay at some point we need to try and add unit tests (at least minimally) or may be golden tests to the tasks to make sure some one accidentally does not break it.

**Comment by erwinmombay** [comment]
a golden test would be great, but writing unit tests for these aren't worth it IMO.

I'll add a golden test.

File: build-system/tasks/index.js [hunk 3]
```diff
@@ -20,10 +20,11 @@ require('./clean');
...
 require('./compile-access-expr');
 require('./dep-check');
+require('./get-zindex');
 require('./lint');
 require('./make-golden');
 require('./presubmit-checks');
 require('./serve');
 require('./size');
-require('./test');
+require('./runtime-test');
```
Comment (erwinmombay) [inline build-system/tasks/index.js]: @cramforce fyi, renamed file.

File: build-system/tasks/get-zindex/test.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+
+var fs = require('fs');
+var test = require('ava');
+var module = require('./');
+
+var result = '{"test.css":{".selector-1":"1",' +
+    '".selector-2":"0",".selector-3":"99"}}';
+
+test.cb('collects selectors', t => {
```
Comment (erwinmombay) [inline build-system/tasks/get-zindex/test.js]: @sriramkrish85 tests added here

**Pull request merged**